mne/viz/_3d.py

```diff
@@ -2187,8 +2187,9 @@ def link_brains(brains, time=True, camera=False, colorbar=True, picking=False):
 
     if _get_3d_backend() != "pyvistaqt":
         raise NotImplementedError(
-            "Expected 3d backend is pyvistaqt but"
-            " {} was given.".format(_get_3d_backend())
+            "Expected 3d backend is pyvistaqt but" " {} was given.".format(
```

Comment:
Things like this should be manually edited.

Reply:
Two strings on the same line?

Reply:
Oh my! It would be good to fix those. I get 442 results on main, so if necessary I'll manually check all of them.